server/services/interventionsService.test.ts

```diff
@@ -191,22 +191,27 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
 
   describe('createDraftReferral', () => {
     it('returns a newly created draft referral', async () => {
+      const interventionId = '24b8cd48-ccb2-48e0-a842-f5673c73e38f'
+      const serviceUserCRN = 'X862134'
+
       await provider.addInteraction({
-        // We're not sure what is an appropriate state here
-        state: 'a draft referral can be created',
+        state: 'an intervention has been selected and a draft referral can be created',
         uponReceiving: 'a POST request to create a draft referral',
         withRequest: {
           method: 'POST',
           path: '/draft-referral',
           headers: { Accept: 'application/json', Authorization: `Bearer ${token}` },
-          body: { serviceUserCrn: 'X862134' },
+          body: {
+            serviceUserCrn: serviceUserCRN,
+            interventionId,
```

Comment:
this should also be a named field e.g. `interventionId: interventionId`

Reply:
This is JS property shorthand for the above as the key and variable name are the same (the linter complains otherwise, but I can rename the variable if you find it clearer, or we can turn off that linting rule?) I'll change it for the one above too if you're happy with it being shorthand (doesn't complain as `CRN` vs Crn`)

Reply:
Have updated this to use property shorthand in both cases, but lemme know if you'd rather we didn't.

Reply:
oh cool, awesome thanks